webpack/es-light.config.js

```diff
@@ -0,0 +1,17 @@
+const { merge } = require('webpack-merge');
+const webpackConfig = require('./es6.config');
+const path = require('path');
+
+delete webpackConfig.entry; // overwrite
```

Comment:
could be that there is something in webpack-merge api that allows you to overwrite when merging which maybe can be easier to read as it is more explict ? (Just a thought, i did not really go and checked their API)

Reply:
I thought `delete webpackConfig.entry` would be as explicit as it gets, what could be more explicit than that? :)